.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+## Description
+
+## Info Required
+
+- Which version of the library do you actually use?
+
+- Do you have the localization permission granted?
```

Comment:
Maybe i will use Checkboxes for that:

[ ] Do you have the localization permission granted?